source/parser/util.civet

```diff
@@ -459,6 +460,8 @@ skipParens := new Set [
  * (Consider renaming and making parentheses depend on context.)
  */
 function makeLeftHandSideExpression(expression: ASTNode)
+  while [ item ] := expression
```

Comment:
Should we check `isArray` and `length is 1` as well?

Reply:
You [already did that](https://civet.dev/playground?code=d2hpbGUgWyBpdGVtIF0gOj0gZXhwcmVzc2lvbg%3D%3D). 😺 

```js
while [ item ] := expression
↓↓↓
function len<
  T extends readonly unknown[],
  N extends number,
>(arr: T, length: N): arr is T & { length: N } {
  return arr.length === length;
}
while (
  expression &&
  Array.isArray(expression) &&
  len(expression, 1)
) {
  const [item] = expression;
}
```

Reply:
Sometimes I spend too much time in JS land and I forget the magic of Civet :cat: